_opensearch/rest-api/snapshots/get-snapshot-repository

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Get Snapshot Repository
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 3
+---
+
+## Get snapshot repository.
+
+Retrieves information about a snapshot repository.
+
+To learn more about repositories, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
+
+### Path parameters
+
+| Parameter | Data Type | Description |
+| :--- | :--- | :--- |
+| repository | String | A comma-separated list of snapshot repository names to retrieve. Wildcard (`*`) expressions are supported including combining wildcards with exclude patterns starting with `-`. |
+
+### Query parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| local | Boolean | Whether to get information from the local node. Optional, defaults to `false`.|
+| master_timeout | Time | Amount of time to wait for a connection to the master node. Optiona, defaults to 30 seconds. |
```

Comment:
Does this parameter still have this name? We change other master_timeout parameter names to cluster_manager_timeout.

Reply:
@kolchfa-aws I just ran the API like this: `GET /_snapshot/my-opensearch-repo?master_timeout=10s` and no errors were returned. Also, in the dev console, `master_timeout` appeared as a suggestion (see attachment).

I also ran it like this: `GET /_snapshot/my-opensearch-repo?cluster_manager_timeout=0s`, with the same output as the previous command.

I'm not sure how to handle this. Please advise. Thanks!

<img width="763" alt="Screen Shot 2022-08-18 at 10 15 54 AM" src="https://user-images.githubusercontent.com/30219968/185444654-f863397e-b2e5-4471-8dfd-a6f071f653cc.png">

Reply:
Please use cluster_manager_timeout as this is in line with our inclusive language guidelines. Thank you!

Reply:
Done.